### PR TITLE
feat: bluez

### PR DESCRIPTION
**Pull request by wash2**
This is a start at trying to create a usable bluez crate for applets and settings. I have not tested it much, so i can't guarantee that it works yet but I'd like to try using it in a bluetooth applet soon as a test. 

As far as I can tell, it is a bit tricky to get the adapters and devices. the default adapter would probably be ok to assume in most cases, but i assume it might be best to just search for the adapter in managed objects instead. 

It seems that getting the device list requires a similar approach as well.

### CODE REVIEW

File: bluez/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "bluez"
```
Comment (mmstick) [inline bluez/Cargo.toml:2]: Should be `bluez-zbus`

**Review by mmstick — CHANGES_REQUESTED** [review]

**Comment by mmstick** [comment]
[bluez/bluer](https://github.com/bluez/bluer) seems to be an official bluez binding for Rust from bluez. Though it interfaces with bluez via C bindings.

**Comment by mmstick** [comment]
bluer has an open issue seeking PRs for zbus support, so we could contribute this there.

**Comment by wash2** [comment]
ok i'll probably close this then and look into possibly contributing this to bluer instead :)

**Pull request closed without merging**